proxy/http/HttpCacheSM.cc

```diff
@@ -159,7 +159,8 @@ HttpCacheSM::state_cache_open_write(int event, void *data)
 {
   STATE_ENTER(&HttpCacheSM::state_cache_open_write, event);
   ink_assert(captive_action.cancelled == 0);
```

Comment:
Please reinit the captive_action. Otherwise this action will be dirty by `pending_action->cancel` in HttpSM.

Reply:
Fixed. Thanks